app/scripts/services/dimStoreService.factory.js

```diff
@@ -8,24 +8,23 @@
 
   function StoreService($rootScope, $q, dimBungieService, dimPlatformService, dimItemTier, dimCategory, dimItemDefinitions, dimBucketService, dimStatDefinitions, dimObjectiveDefinitions, dimTalentDefinitions, dimSandboxPerkDefinitions, dimYearsDefinitions, dimProgressionDefinitions, dimRecordsDefinitions, dimInfoService, SyncService, loadingTracker) {
     var _stores = [];
-    var progressionDefs = {};
-    let recordsDefs = {};
-    var buckets = {};
-    var idTracker = {};
+    var _progressionDefs = {};
```

Comment:
They're to indicate global state. We were using them inconsistently, so I just moved them all over.